deploy/helm/sumologic/values.yaml

```diff
@@ -261,7 +261,7 @@ sumologic:
 fluentd:
   image:
     repository: public.ecr.aws/sumologic/kubernetes-fluentd
-    tag: 1.14.4-sumo-1
+    tag: 1.14.5-sumo-1
```

Comment:
This needs to be in a separate PR / commit with changes to support matrix as well.